src/wymeditor/editor/base.js

```diff
@@ -2189,15 +2083,27 @@ WYMeditor.editor.prototype.insert = function (html) {
 WYMeditor.editor.prototype.wrap = function (left, right) {
```

Comment:
Simple changes in `wrap` and `unwrap`.

BTW, they are quite buggy, these methods. They don't always get what the user intended.

Luckily, the only place I've noticed use of them, is in the code of one of the examples.